governance.md

```diff
@@ -109,6 +110,16 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+Working groups do not own any code or subprojects. Instead, they are a place for people to discuss topics that cross SIG boundaries.
+
+Working groups are primarily used to facilitate topics of discussion that are in scope for Kubernetes but that cross SIG lines.
+If a set of folks in the community want to get together and discuss a topic, they can do so without forming a Working Group.
```

Comment:
It seems like this paragraph and the first paragraph should be merged or refactored. 

I also want to clarify the "short-lived" aspect. We expect WGs to not live forever -- they should have a goal and concrete deliverables, but being short-lived shouldn't be a reason to create a WG. For example, a WG isn't needed for every short-term effort. Every release ships the results of many short-term efforts.

I also do want to address the class of SIGs/WGs for "use case X on Kubernetes": most of SIG Apps, SIG Big Data, M L WG, the proposed IoT/Edge WG. I will send email about that.

Most of our current WGs probably should not be WGs.